docs/explanation/adr/0017-jvm-jar-distribution-strategy.md

```
@@ -0,0 +1,537 @@
+    mergeServiceFiles()
+}
+
+// Remove GraalVM plugin and configuration
+// plugins {
+//     alias(libs.plugins.graalvm.native)
+// }
+```
+
+### Distribution Package Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)